homeworks/karolinavalko/2_vars_datatypes/user_info.py

```diff
@@ -0,0 +1,96 @@
+from os import system
+
+
+from pprint import pprint
+
+system("cls")
+
+"""
+1. Kérj be a felhasználótól 4 programozási nyelvet vesszővel elválasztva, szóközök nélkül. 
+2. Konvertáld a kapott stringet egy listává, és add hozzá a fenti dictionary-hez “skills” néven.
+3. Rendezd a favourite_meals lista elemeit abc szerinti növekvő sorrendbe.
+    Printeld ki a favourite_meals lista utolsó előtti elemét
+4. Adj hozzá egy “spaghetti” string-et ugyanehhez a listához.
+5. Add hozzá a favourite_meals-hez az aktuális favourit_meals lista harmadik és negyedik elemét (nem az index-ét) újra.
+6. Ezután töröld az így keletkezett duplikátumokat!
+7. Cseréld fel a favourite_meals lista első és utolsó elemét!
+8. A “phone_contacts” dictionary-hez adj hozzá egy új elemet, tetszőleges névvel és telefonszámmal.
+9. Tim és Tim2 ugyanazt az embert reprezentálják a “phone_contacts”-ban, viszont a "Tim" key mögött lévő 
+telefonszám már nem él. Töröld ki a telefonkönyvből!
+10. Adj hozzá egy olyan új embert “phone_contacts”-hoz, akinek 2 telefonszáma is van!
+
+Extra 1: Printeld ki a “skills”  lista utolsó 3 elemét ellentétes sorrendben!
+Extra 2: Most, hogy Tim-nek már csak 1 telefonszáma van, érdemes lenne átnevezni Tim2-t Tim-re!
+"""
+
+print("\nAnswer for the first question")
+programming_languages = input(
+    "Add 4 programming language types with comma separated: "
+).split(",")
+
+user_info = {
+    "name": "Mike",
+    "age": 25,
+    "favourite_meals": ["pizza", "carbonara", "sushi"],
+    "phone_contacts": {
+        "Mary": "+36701234567",
+        "Tim": "+36207654321",
+        "Tim2": "+36304567321",
+        "Jim": "+364005000",
+    },
+}
+
+user_info["skills"] = programming_languages
+pprint(user_info["skills"])
+
+
+print("\nAnswer for the second and third questions")
+user_info["favourite_meals"] = sorted(user_info["favourite_meals"])
```

Comment:
ez is szuper megoldás, nem javítani való csak egy komment:
mivel a dictionary mutable, ezért "in-place" tudod sortolni a .sort() method-dal is, és akkor nem kell újra assign-olni.
Tehát ehelyett:
user_info["favourite_meals"] = sorted(user_info["favourite_meals"])

elég az is hogy
user_info["favourite_meals"].sort()